uber/configspec.ini

```diff
@@ -446,6 +446,13 @@ guest_badge     = string(default="Guest")
 one_day_badge   = string(default="One Day")
 __many__ = string
 
+[[badge_status]]
```

Comment:
These are the statuses that every convention will definitely use.

I'm conflicted on whether or not to include a `printed_status` variable here - it was used a bunch for the logic in Anthrocon, but should it be included in core even though not every convention will need it as a status? Hm.